### PR TITLE
Enable payload continue

### PR DESCRIPTION
**Pull request by biryukovmaxim**

### CODE REVIEW

File: consensus/benches/check_scripts.rs [hunk 1]
```diff
@@ -116,11 +129,31 @@ fn benchmark_check_scripts(c: &mut Criterion) {
...
+        });
+    }
+}
+
 criterion_group! {
     name = benches;
-    // This can be any expression that returns a `Criterion` object.
     config = Criterion::default().with_output_color(true).measurement_time(std::time::Duration::new(20, 0));
-    targets = benchmark_check_scripts
+    targets = /*benchmark_check_scripts,*/ benchmark_check_scripts_with_payload
```
Comment (someone235) [inline consensus/benches/check_scripts.rs]: Don't leave redundant comments

File: consensus/core/src/hashing/tx.rs [hunk 2]
```diff
@@ -157,6 +157,13 @@ mod tests {
             expected_hash: "31da267d5c34f0740c77b8c9ebde0845a01179ec68074578227b804bac306361",
         });
 
+        // Test #8, same as 7 but with a non-zero payload. The test checks id and hash are affected by payload change
```
Comment (someone235) [inline consensus/core/src/hashing/tx.rs:160]: Please add similar test to `test_signature_hash`

File: testing/integration/src/consensus_integration_tests.rs [hunk 3]
```diff
@@ -1903,3 +1904,114 @@ async fn payload_test() {
 
     consensus.shutdown(wait_handles);
 }
+
+#[tokio::test]
+async fn run_payload_activation_test() {
```
Comment (someone235) [inline testing/integration/src/consensus_integration_tests.rs]: There's no need for `run_` in test names

**Review by someone235 — CHANGES_REQUESTED** [review]

**Review by someone235 — CHANGES_REQUESTED** [review]
Also, remove from `consensus/core/src/hashing/sighash.rs` comment saying `// TODO: Right now this branch will never be executed, since payload is disabled`

File: consensus/core/src/hashing/tx.rs (hunk 2 above)
Comment (biryukovmaxim) [reply consensus/core/src/hashing/tx.rs:160]: commit 03a8ca88ec38928f1183d1207bd8a30b8146da80

File: testing/integration/src/consensus_integration_tests.rs (hunk 3 above)
Comment (biryukovmaxim) [reply testing/integration/src/consensus_integration_tests.rs]: commit 2df77655270c6ea6c0d3ed52a689a5224a455662

File: consensus/benches/check_scripts.rs (hunk 1 above)
Comment (biryukovmaxim) [reply consensus/benches/check_scripts.rs]: It's already been removed

**Comment by biryukovmaxim** [comment]
> Also, remove from `consensus/core/src/hashing/sighash.rs` comment saying `// TODO: Right now this branch will never be executed, since payload is disabled`

It's already been removed

**Review by someone235 — APPROVED** [review]

**Pull request merged**